src/cider/nrepl/middleware/track_state.clj

```diff
@@ -43,6 +43,12 @@
     (:fn-var m) (assoc :fn true)))
 
 ;;; Namespaces
+(defn- relevant-meta-keys
+  "Access orchard.meta/relevant-meta-keys. For use in test cases where
+  direct access to the symbol is not possible."
+  []
+  m/relevant-meta-keys)
```

Comment:
I don't think this function is useful outside `cider-nrepl`, so we should probably bring it back. Might have missed a few of those when working in bulk.

Reply:
If a recall correctly at some point our bencode implementation was failing to encode certain complex metadata, so we decided to circumvent this by just selecting the metadata needed by CIDER instead of retrieving all metadata. This means that such a function would not be useful outside `cider-nrepl` most likely.